commit 3f0dcbf51826ce30878b3f9335fe8499325816a9
Author: Dan Oscarsson <DanOscarsson@users.noreply.github.com>
Date:   Fri Apr 28 12:02:50 2017 +0200

sub: sdh filter fix for null

sdh filter returns NULL when subtitle line should be skipped

diff --git a/sub/sd_ass.c b/sub/sd_ass.c
index 64051337f..e81583c76 100644
--- a/sub/sd_ass.c
+++ b/sub/sd_ass.c
@@ -283,7 +283,7 @@ static void decode(struct sd *sd, struct demux_packet *packet)
         int ass_len = packet->len;
         if (sd->opts->sub_filter_SDH) {
             ass_line = filter_SDH(sd, track->event_format, 1, ass_line, ass_len);
-            ass_len = strlen(ass_line);
+            ass_len = ass_line ? strlen(ass_line) : 0;
         }
         if (ass_line)
             ass_process_chunk(track, ass_line, ass_len,
